survival/km.py

```diff
@@ -0,0 +1,187 @@
+import pandas as pd
+import numpy as np
+import matplotlib.pyplot as plt
+import copy
+from survival.utils import intersect_df, assign_quantiles, intersect_series_with_index
+from plotting.utils import generate_color_palette
+
+from lifelines.plotting import add_at_risk_counts
+from lifelines.statistics import logrank_test
+from lifelines import KaplanMeierFitter
+
+class SurvivalInfo:
```

Comment:
take SurvivalInfo object to another python file out of km.py.
maybe called survival.survival_info.py